tools/ou3_p5_45deg_sample1_entry_ha_v2.py

```
@@ -0,0 +1,123 @@
+  'cold_lock_follows_with_mag':'accel_bias_locked_   = with_mag_;' in w,
+  'live_enable_requires_lock_cleared':'const bool allow_bias = !accel_bias_locked_ && !acc_bias_hold_;' in w,
+  'mag_unlock_requires_update_count':'mag_updates_applied_ >= mag_updates_to_unlock_' in w,
+  'mag_unlock_has_one_second_guard':re.search(r'time_\)\s*-\s*first_mag_update_time_\)\s*>\s*1\.0f',w) is not None,
+  'deployed_unlock_count_250':'int acc_bias_unlock_mag_updates = 250;' in w and 'MAG_UPDATES_TO_UNLOCK = 250' in w,
+  'shipping_projection_is_radial_closed_ball':'if (n > acc_bias_limit_)' in k and 'b *= (acc_bias_limit_ / n);' in k,
+ }
+ return checks
+
+
+def _acc_child_projection(P,e,mode,H,R,rho,dcap,ba_cap,proj):
+ PHt,S=V1.FULL._innovation(P,H,R); Sinv,b=V1.FULL._spd_inverse_enclosure(S,R); K=V1.FULL.matrix_mul(PHt,Sinv); dx=V1.FULL._mat_vec(K,V1.FULL._vec_box(rho)); caps=V1._caps(PHt,P,R,mode,rho); dx2=list(dx)
```

Ruff (0.16.3)

[error] Multiple statements on one line (semicolon)

(E702)